example/lib/main.dart

```diff
@@ -122,48 +121,22 @@ class _MyAppState extends State<MyApp> {
     });
   }
 
-  Future<void> provisionAbly() async {
-    setState(() {
-      _provisioningState = OpState.inProgress;
-    });
-
-    provisioning.AppKey appKey;
-    try {
-      appKey = await provisioning.provision('sandbox-');
-      print('App key acquired! `$appKey`');
-    } on Exception catch (error) {
-      print('Error provisioning Ably: $error');
-      setState(() {
-        _provisioningState = OpState.failed;
-      });
-      return;
-    }
-
-    setState(() {
-      _appKey = appKey;
-      _provisioningState = OpState.succeeded;
-    });
-  }
-
   Future<void> createAblyRest() async {
     setState(() {
       _restCreationState = OpState.inProgress;
     });
 
-    final clientOptions = ably.ClientOptions()
-          // .fromKey(_appKey.toString())
-          ..environment = 'sandbox'
+    final clientOptions = ably.ClientOptions.fromKey(_apiKey)
           ..logLevel = ably.LogLevel.verbose
           ..logHandler = ({msg, exception}) {
             print('Custom logger :: $msg $exception');
           }
-          ..tokenDetails = ably.TokenDetails.fromMap(
+/*          ..tokenDetails = ably.TokenDetails.fromMap(
```

Comment:
The example code needs to be authoritative so commented out sections generally aren't helpful, especially if they've not got some kind of meta-comment next to them that explains _why_ they're commented out or what re-introducing them might do.

Reply:
Thanks for catching that. I left it in that state when I wasn't sure if we should remove the provisioning code from the example app. We decided that it was okay because users would never need to provision an Ably account themselves. Commented code removed in 7fd639f7ae5392e00babb7148cae9f16bd86a5fa